docs/source/keylime/index.rst

```diff
@@ -0,0 +1,135 @@
+Keylime
+=======
+
+ESI provides a guide to use `Keylime`_ outside Ironic for run-time attestation with `IMA`_.
+
+Prerequisites
+-------------
+* A private attestation network.
+* A node specified as Keylime server.
+* The nodes-to-be-attested with an active TPM 2.0 and keylime agent installed.
+
+Setup
+-----
+
+Build Keylime Server
+~~~~~~~~~~~~~~~~~~~~
+Provision the node:
+
+.. prompt:: bash $
+
+  metalsmith deploy --image <image> --network <network> --resource-class baremetal --ssh-public-key <path_to_public_key>
```

Comment:
I'd specify <attestation_network> here to make it clear where the node needs to end up.